src/components/Read.js

```diff
@@ -19,12 +19,20 @@ const Read = ({ letter, handleClick, isWrite }) => {
             <Icon icon={letter.icon} />
           </span>
         </div>
-        {isWrite ? null : (
-          <Button onClick={handleClick} className='ui button'>
+      </div>
+      {isWrite ? null : (
+        <>
+          <Button
+            onClick={() => setDesk(true, true, false)}
+            className='ui right floated button'
```

Comment:
Since we are doing this string of classes multiple times, we should make a variable name for it